Framework/Foundation/include/Framework/Pack.h

```diff
@@ -272,6 +272,28 @@ constexpr auto unique_pack(pack<T, Ts...>, PT p2)
 template <typename P>
 using unique_pack_t = decltype(unique_pack(P{}, pack<>{}));
 
+template <typename... Ts>
+inline constexpr std::tuple<Ts...> pack_to_tuple(pack<Ts...>)
+{
+  return std::tuple<Ts...>{};
+}
+
+template <typename P>
+using pack_to_tuple_t = decltype(pack_to_tuple(P{}));
+
+template <typename T, std::size_t N, typename... REST>
+constexpr auto repeat_type_pack(pack<REST...>)
```

Comment:
Why do you need to pass the pack?

Reply:
right, perhaps I can get rid of this, let me try.